<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataRiverSurveyCrosssectionWindow</class>
 <widget class="QMainWindow" name="GeoDataRiverSurveyCrosssectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="hSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GeoDataRiverSurveyCrosssectionWindowGraphicsView" name="graphicsView" native="true"/>
      <widget class="QSplitter" name="vSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="surveysTableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="GeoDataRiverSurveyCrosssectionWindowWseTableWidget" name="wsesTableWidget"/>
       <widget class="GeoDataRiverSurveyCrosssectionWindowTableView" name="tableView"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="crossSectionToolBar">
   <property name="windowTitle">
    <string>Cross Section ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>View Setting ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="displayToolBar">
   <property name="windowTitle">
    <string>Display Setting ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeoDataRiverSurveyCrosssectionWindowGraphicsView</class>
   <extends>QWidget</extends>
   <header>geodatariversurveycrosssectionwindowgraphicsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeoDataRiverSurveyCrosssectionWindowTableView</class>
   <extends>QTableView</extends>
   <header>geodatariversurveycrosssectionwindowtableview.h</header>
  </customwidget>
  <customwidget>
   <class>GeoDataRiverSurveyCrosssectionWindowWseTableWidget</class>
   <extends>QTableWidget</extends>
   <header>geodatariversurveycrosssectionwindowwsetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
